doc/conventions/others.rst

```diff
@@ -48,10 +48,10 @@ Loss is implemented by a CPTP map whose Kraus representation is
 .. admonition:: Definition
     :class: defn
 
-    Loss is implemented by coupling mode :math:`\a` to another bosonic mode prepared in the vacuum state, by using the following transformation
+    Loss is implemented by coupling mode :math:`\a` to another bosonic mode :math:`\hat{b}` prepared in the vacuum state, by using the following transformation
```

Comment:
add hat to a

Reply:
\a is already defined as :math:`\hat{a}` :)